python/mxnet/gluon/nn/basic_layers.py

```diff
@@ -124,6 +124,12 @@ class Dense(HybridBlock):
         (ie. "linear" activation: `a(x) = x`).
     use_bias : bool
         Whether the layer uses a bias vector.
+    last_axis: bool
+        Whether the dot and addition are applied on the last axis.
+        If true, all but the last axis of input data are kept the same, and the transformation
+        applies on the last axis.
+        If false, all but the first axis of input data are multiplied together to become the
```

Comment:
Actually, let's call this option flatten since by default it's the same with flatten.